Src/CSharpier.Tests/FormattingTests/TestFiles/InvocationExpressions.cst

```diff
@@ -197,21 +197,17 @@ class ClassName
         // comment shouldn't do weird stuff
         SomeObject.CallMethod();
 
-        CallMethod(
-                lambdaWithBlock =>
-                {
-                    return;
-                }
-            )
+        CallMethod(lambdaWithBlock =>
+            {
+                return;
+            })
             .CallMethod()
             .CallMethod();
 
-        someObject.CallMethod(
-            lambdaWithBlock =>
-            {
-                return;
-            }
-        );
+        someObject.CallMethod(lambdaWithBlock =>
```

Comment:
Is this really "better"? IMHO the previous version was more readable for both these cases.

Reply:
I think it is mostly better, with those examples being the ones that fall into "maybe not better" category.

These examples I think are definitely better. It saves vertical space and I don't think it is any less readable there.
https://github.com/belav/csharpier-repos/pull/41/commits/831f2012d4b99fa4f5cbdec68b6c2cb988a71fce#diff-bc2e3864cdf7b47541cca4aa46b6d378cb2c11cb0091dca0647fed8a565a7dbf

It indents less too, which helps in other cases like this
https://github.com/belav/csharpier-repos/pull/41/commits/831f2012d4b99fa4f5cbdec68b6c2cb988a71fce#diff-668431d269a33a846504a3895822785838e4d92a10e9a3ce025b37de65724ed2

fwiw, prettier formats it like this. I don't really like that they force `( )` on it
```js
someObject.CallMethod((lambdaWithBlock) => {
    return;
});